contracts/BadAuth.sol

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Peter Robinson 2023
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+interface FlashLoanReceiverInterface {
+   function flashLoanReceiver(uint256 _fee, bytes calldata _params) external payable;
+}
+
+/**
+ */
+abstract contract BadAuth {
+    uint256 constant INTEREST_DIVISOR = 1000000;
+
+    // Minimum period a deposit must be held, in blocks.
+    uint256 constant MIN_HOLD_PERIOD = 1000;
+
+    // Minimum number of blocks between calling setInterestRate and changeInterestRate.
```

Comment:
Once we load the contracts, we should remove comments, as they might influence the LLM. What do you think?

Reply:
Good point (for testing). I added a `--without-comments` flag, since when actually using the cli tool on our own contracts, I guess we should keep the comments for optimal results (therefore  --without-comments defaults to False).